src/github.rs

```diff
@@ -403,16 +375,28 @@ impl RepositoryClient<'_> {
                 .without_body()
                 .send_with_response::<Vec<CommitData>>()?;
             for commit in commits {
-                if commit.parents.iter().any(|p| p.sha == merged) {
+                self.start_new_request()?;
+                self.client.get(true)?;
+                self.client.url(&format!(
+                    "https://api.github.com/repos/{repo}/commits/{sha}",
+                    repo = self.repo,
+                    sha = commit.sha,
+                ))?;
+                let commit = self
+                    .client
+                    .without_body()
+                    .send_with_response::<FullCommitData>()?;
+                if commit.files.iter().any(|f| f.filename == path) {
```

Comment:
This will result in a ton of API requests, but we can fix this once the release is done.